cmd/juju/model/cancelgeneration.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package model
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/modelgeneration"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const (
+	cancelGenerationSummary = "Cancels the new generation to the model."
+	cancelGenerationDoc     = `
+Cancel the next generation. This will abort anything in the next
```

Comment:
there is a card to refine the help text here: https://trello.com/c/VBlUfGju/925-refine-text-for-cancel-generation-cmd-help